bl/handlers.go

```diff
@@ -0,0 +1,147 @@
+package bl
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/YuvalSteiy/book_service/dal"
+	"github.com/YuvalSteiy/book_service/models"
+	"github.com/gin-gonic/gin"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+)
+
+func GetBookByID(c *gin.Context) {
+	updateUserData(c, "GET")
+	db, err := dal.NewBookStorer()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, err)
+		return
+	}
+	id := c.Param("id")
```

Comment:
id is a URL parameter. if we "GET /book" without specifying the id we would get-
"404 page not found" since such route is not defined.
In my opinion it's ok since "get book by id" without id should not work. What do you think?